ostelco-ios-client/ViewControllers/Login/LoginViewController.swift

```diff
@@ -108,7 +107,12 @@ class LoginViewController2: UIViewController {
     }
     
     @objc private func showEKYCOhNo() {
-        performSegue(withIdentifier: "showEKYCOhNo", sender: nil)
+        let ohNo = OhNoViewController.fromStoryboard(type: .ekycRejected)
+        ohNo.primaryButtonAction = {
+            // TODO: What are we actually retrying when this fails?
```

Comment:
Retry means start ekyc process again, in practical terms it means send user to ekyc on boarding screen

Reply:
Thanks for the reminder - I meant to update this when I saw it in Slack and then i got distracted 😛